### ISSUE TITLE
Slides with nothing but an image not rendering

### ISSUE THREAD

**Issue by matiasandina** [quarto-dev/quarto-r#20]
I am experiencing some issue with slides that have images included using `knitr::include_graphics()`. Whenever these slides have no title or text, they are left out of the presentation. This doesn't happen if I put any text into that slide (or if I add a title). This doesn't happen with a slide that has no text or title but a regular print chunk (see second slide `head(mtcars)`, which renders fine) I wonder if there's a particular chunk option that I have to select or whether this is a bug.

```
---
title: "Untitled"
format: revealjs
editor: visual
---

{r}
knitr::include_graphics("https://d33wubrfki0l68.cloudfront.net/521a038ed009b97bf73eb0a653b1cb7e66645231/8e3fd/assets/img/rstudio-icon.png")

---

{r}
head(mtcars)

```

This also happens when I add them via typical markdown (using the import widget on Rstudio).
This one won't get included (although it's showing nicely on the visual Rstudio editor)

```
## 

![](img/path_to_img.png){fig-align="center"}

```
This one will render fine!

```
## .

![](img/path_to_img.png){fig-align="center"}

```

**Comment by cderv** [comment]
Can you share a reproducible example ? 

This example renders fine on my side 

````markdown
---
title: "Demo .qmd"
format: revealjs
---

## Head

Content

## 

![](https://revealjs.com/images/logo/reveal-black-text.svg){fig-align="center"}

````

The slide with the image is here.

**Comment by matiasandina** [comment]
I'm sorry, I thought my code was reproducible. Anyhow, I guess the problem is on my side then, because I copied and pasted your code into an empty `.qmd` and it has the same exact issue than my example. Second slide is not rendering.

This is the Rstudio I'm using.

```
"Prairie Trillium" Release (9f796939, 2022-02-16) for Ubuntu Bionic
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36
```

```
~$ quarto -v
0.9.290
```

**Comment by cderv** [comment]
This issue may have been fix since the 0.9.290 version. I am on very last development version and latest build is https://github.com/quarto-dev/quarto-cli/releases/tag/v0.9.655 

So you should update quarto. 

You could also install last RStudio IDE Preview version which bundle a greater version

**Comment by matiasandina** [comment]
Perfect, updating quarto works
